resources/schema/types.py

```diff
@@ -218,28 +223,22 @@ def resolve_piers(self, info, **kwargs):
         return resolve_piers(info, **kwargs).filter(harbor_id=self.id)
 
     def resolve_max_width(self, info, **kwargs):
```

Comment:
max_width and max_length changes quite rarely, could be cached if it is a performance issue?

Reply:
Hmmm good point, we can look into caching later tho.